src/store/index-level.ts

```diff
@@ -1,213 +1,284 @@
-import type { Filter, RangeFilter } from '../types/message-types.js';
+import type { FilteredQuery } from '../types/event-log.js';
+import type { LevelWrapper } from './level-wrapper.js';
+import type { EqualFilter, Filter, OneOfFilter, RangeFilter } from '../types/message-types.js';
 import type { LevelWrapperBatchOperation, LevelWrapperIteratorOptions } from './level-wrapper.js';
 
-import { executeUnlessAborted } from '../index.js';
 import { flatten } from '../utils/object.js';
-import { createLevelDatabase, LevelWrapper } from './level-wrapper.js';
+import { SortOrder } from '../types/message-types.js';
+
+
+type IndexedItem<T> = {
+  value: T;
+  key: string;
```

Comment:
Assuming I understand correctly, please rename to `id`. Got confused the `key` being a index key.

Reply:
#572 submitted for review.